flypipe/cache/cache_operation.py

```diff
@@ -0,0 +1,5 @@
+from enum import Enum
+
+
+class CacheOperation(Enum):
```

Comment:
changed `.run(cache({"disable": [node1]}))` to `.run(cache({node1: CacheOperation.DISABLE}))`